_episodes/02-practice-learning.md

```diff
@@ -75,61 +75,99 @@ tasks with normal effort under normal circumstances.
     > Example: An *expert* in a Carpentries workshop may have experience writing and running shell scripts and, when
     presented with a problem, immediately sees how these skills can be used to solve the problem.
 
+Note that, while you may not think of yourself as a computational "expert," in the context above that term may accurately describe
+your skill level. We will come back to the expertise of the instructor and its impact -- positive and negative -- on teaching, in the next episode. 
 For now, we are primarily concerned with novices,
-as this tends to characterize The Carpentries audience. (We, as instructors,
-  are more likely to be competent practitioners or experts, and we will
-discuss this in a later section).  We will next use an additional concept
-to help us differentiate between novices and competent practitioners, which will have
-implications for how we teach novices.
-
-## Cognitive Development and Mental Models
-
-Effective learning is facilitated by the creation of a well-founded mental model. A mental model is a collection of concepts and facts,
-along with the relationships between those concepts, which a person has about a topic. For example, a long-time resident of the United
-States may have an advanced understanding of the location of US states, major cities and landmarks, weather patterns, regional
-economies and demographic patterns, as well as the relationships among these, compared with their understanding of these relationships
-for other countries. In other words, their mental model of the United States is more complex compared with their mental model of other
-countries.
-
-We can distinguish between a *novice* and a *competent
-practitioner* for a given domain based on the complexity of their mental models.
-
-*     A *novice* is someone who has not yet built a mental model of the domain.
-They therefore reason by analogy and guesswork,
-borrowing bits and pieces of their mental models of other domains
+as this tends to characterize The Carpentries audience.
```

Comment:
```suggestion
as this is The Carpentries target audience.
```

I vote for slighly stronger here in that this is the vision and that it's not a coincidence but we sometimes get others.  Carpentries materials are designed for novices and we have strategies to accomodate others to varying degrees.

Reply:
```suggestion
as this is The Carpentries' primary target audience.
```
Agree; adding "primary" to be inclusive (and some lessons are more novice than others)